test/plots/scatterPlotTests.ts

```diff
@@ -2,395 +2,386 @@
+        plot.renderTo(svg);
+        let changedMetadata = { foo: 0, bar: 0 };
+        dataset.metadata(changedMetadata);
 
-    it("can retrieve entities in a certain bounds", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
-
-      let dataset = new Plottable.Dataset([{x: 0, y: 0}, {x: 1, y: 1}]);
-      let dataset2 = new Plottable.Dataset([{x: 1, y: 2}, {x: 3, y: 4}]);
-      let plot = new Plottable.Plots.Scatter();
-      plot.x((d: any) => d.x, xScale)
-          .y((d: any) => d.y, yScale)
-          .addDataset(dataset)
-          .addDataset(dataset2);
-      plot.renderTo(svg);
-
-      let entities = plot.entitiesIn({ topLeft: {
-                                         x: xScale.scale(1),
-                                         y: yScale.scale(1)
-                                       },
-                                       bottomRight: {
-                                         x: xScale.scale(1),
-                                         y: yScale.scale(1)
-                                       }});
-
-      assert.lengthOf(entities, 1, "only one entity has been retrieved");
-      assert.deepEqual(entities[0].datum, {x: 1, y: 1}, "correct datum has been retrieved");
-
-      svg.remove();
-    });
+        let symbols = plot.selections();
+        assert.strictEqual(symbols.size(), 2, "exactly 2 symbols are rendered");
 
-    it("can retrieve entities centered at a given point", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
+        let c1 = d3.select(symbols[0][0]);
+        let c2 = d3.select(symbols[0][1]);
+        let c1Position = d3.transform(c1.attr("transform")).translate;
+        let c2Position = d3.transform(c2.attr("transform")).translate;
+        assert.closeTo(c1Position[0], 0, 0.01, "first symbol cx is correct after metadata change");
+        assert.closeTo(c1Position[1], 0, 0.01, "first symbol cy is correct after metadata change");
+        assert.closeTo(c2Position[0], 1, 0.01, "second symbol cx is correct after metadata change");
+        assert.closeTo(c2Position[1], 0, 0.01, "second symbol cy is correct after metadata change");
 
-      let dataset = new Plottable.Dataset([{x: 0, y: 0}, {x: 1, y: 1}]);
-      let plot = new Plottable.Plots.Scatter();
-      plot.x((d: any) => d.x, xScale)
-          .y((d: any) => d.y, yScale)
-          .addDataset(dataset);
-      plot.renderTo(svg);
+        svg.remove();
+      });
 
-      let entities = plot.entitiesAt({ x: xScale.scale(1), y: yScale.scale(1) });
-      assert.lengthOf(entities, 1, "only one entity has been retrieved");
-      assert.deepEqual(entities[0].datum, {x: 1, y: 1}, "correct datum has been retrieved");
+      it("sets and gets size() correctly", () => {
```

Comment:
I was suggested that testing the getter is useless as we do that everywhere anyway. Amend the test message to `sets the size() correctly`